### ISSUE TITLE
function or associated item `jni_invalid_value` not found for this

### ISSUE THREAD

**Issue by JoyceBabu** [Dushistov/rust_swig#271]
The following code compiles successfully

```
foreign_class!(class Festival {
    fn Festival::test() -> Result<Festival, String>; alias getTest;
});
```

But I cannot return a vector of `Festival` objects. The following code
```
foreign_class!(class Festival {
    fn Festival::test() -> Result<Vec<Festival>, String>; alias getTest;
});
```
fails to compile with error

>  function or associated item `jni_invalid_value` not found for this
> ...
> 24 | # [ allow ( non_snake_case , unused_variables , unused_mut , unused_unsafe ) ] # [ no_mangle ] pub extern "C" fn Java_com_prokerala_calendar_Festival_getTest ( env : * mut JNIEnv , _ : jclass , ) -> internal_aliases :: JForeignObjectsArray < Festival > { let mut ret : Result < Vec < Festival > , String > = Festival :: test ( ) ; let ret : internal_aliases :: JForeignObjectsArray < Festival > = match ret { Ok ( x ) => { let ret : internal_aliases :: JForeignObjectsArray < Festival > = vec_of_objects_to_jobject_array ( env , x ) ; ret } Err ( msg ) => { jni_throw_exception ( env , & msg ) ; return < internal_aliases :: JForeignObjectsArray < Festival > >:: jni_invalid_value ( ) ; } } ; ret }
>    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        ^^^^^^^^^^^^^^^^^ function or associated item not found in `java_glue::internal_aliases::JForeignObjectsArray<model::festival::Festival>`

I checked the jni-test source code, and found that `rust_swig` can accept and return `Vec`. So, why is my code failing? Do I have to include any additional code for vector?

**Comment by JoyceBabu** [comment]
The issue only happens when I return a `Result<Vec<Festival>,String>`.

The following code compiles successfully

```
foreign_class!(class Festival {
    fn Festival::test() -> Vec<Festival>; alias getTest;
});
```

**Comment by Dushistov** [comment]
> But I cannot return a vector of Festival objects. The following code

Thanks for report, this is bug introduced in recent version, should be fixed by #272
